src/services/ldap/index.js

```diff
@@ -1,6 +1,7 @@
 /* eslint-disable no-underscore-dangle */
 const ldap = require('ldapjs');
 const errors = require('@feathersjs/errors');
+const mongoose = require('mongoose');
```

Comment:
Codacy found an issue: [Expected linebreaks to be 'LF' but found 'CRLF'.](https://app.codacy.com/manual/schul-cloud/schulcloud-server/pullRequest?prid=5718258)